### ISSUE TITLE
how much time it takes to generate a simple video?

### ISSUE THREAD

**Issue by lijiansong** [manuelruder/artistic-videos#58]
I run `stylizeVideo.sh` with my own input video, about 10 seconds mp4 video, with GPU `TITAN X (Pascal), 12GB `, 4 hours later, I have't get the output...

**Comment by guillaumebcher** [comment]
Depends on the width/height of your video. Optical flow takes really long time.

**Comment by lijiansong** [comment]
@guillaumebcher yes, when the input is 640x640, it takes nearly 20 hours to get the output video in TITAN X (Pascal), 12GB
